src/net.cpp

```diff
@@ -1068,7 +1068,7 @@ void CConnman::AcceptConnection(const ListenSocket& hListenSocket) {
     SOCKET hSocket = accept(hListenSocket.socket, (struct sockaddr*)&sockaddr, &len);
     CAddress addr;
     int nInbound = 0;
-    int nMaxInbound = nMaxConnections - (nMaxOutbound + nMaxFeeler);
+    int nMaxInbound = nMaxConnections - (nMaxOutbound + nMaxFeeler + nMaxExtraOutbound);
```

Comment:
I'm confused - it looks like we wont ever have both a feeler and an extra outbound open at one time, why change this?

Reply:
> it looks like we wont ever have both a feeler and an extra outbound at one time

We will in general have both a feeler and an extra outbound at the same time.  There was an incorrect comment (from a first iteration of this patch) describing the extra outbound peer as "stealing" the feeler connection, but that is not actually true.  I'll delete that comment.